### PR TITLE
multi-node/generic: basic documentation

### PR DESCRIPTION
**Pull request by pbx0**
First go at some very very basic generic docs. I try to appropriately warn that this is a manual process for when no other setup strategies will do.

@aaronlevy

### CODE REVIEW

**Comment by aaronlevy** [comment]
Thanks for starting on this! We've needed it for a while.

If possible, can we get this down to just the steps one needs to go through?

1.) run tls script like so `foo bar`
2.) copy X assets to /etc/kubernetes/tls on controller, and Y assets to workers.
3.) on controller run /controller.sh on workers run /workers.sh, 

/cc @robszumski

File: multi-node/generic/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Kubernetes on CoreOS Generic Install Scripts
+
+These scripts setup Kubernetes on CoreOS in a similar way to other tools in the repo. The notable difference is that these scripts are intended to be platform agnostic and thus don't automatically setup the TLS assets on each host beforehand. Instead, the contract with these scripts is that first you setup TLS assets on each host manually. Then you copy the `controller-install.sh` to the master node and `worker-install.sh` to any worker nodes. Then you run each script, wait for completetion and you should have a running cluster!
+
+While we provide these scripts and test them through the multi-node Vagrant setup, recommend using a platform specific install method if available. If you are installing to bare-metal, you might find our [baremetal repo](https://github.com/coreos/coreos-baremetal) more appropriate.
```
Comment (dghubble) [inline multi-node/generic/README.md]: typo\recommend\we recommend

**Comment by robszumski** [comment]
I'm gonna take a stab at a rewrite really quick

**Comment by pbx0** [comment]
Closing in favor of #630

**Pull request closed without merging**